types/types.go

```diff
@@ -524,8 +524,6 @@ type SystemContext struct {
 	// Note that this field is used mainly to integrate containers/image into projectatomic/docker
 	// in order to not break any existing docker's integration tests.
 	DockerDisableV1Ping bool
-	// Directory to use for OSTree temporary files
-	OSTreeTmpDirPath string
```

Comment:
This is an API break and it would require a major version bump.

For now, keeping this value around and ignored would work well enough, as far as _this_ type is concerned…

… but, really, dropping an entire subpackage, and the `ostree:` syntax, is an API break and requires a major version bump, either way.

Reply:
I've pushed another patch to bump the version to 6